src/server/qgsserver.cpp

```diff
@@ -61,7 +62,7 @@
 
 QString* QgsServer::sConfigFilePath = nullptr;
 QgsCapabilitiesCache* QgsServer::sCapabilitiesCache = nullptr;
-QgsMapRenderer* QgsServer::sMapRenderer = nullptr;
+QgsMapSettings* QgsServer::sMapSettings = nullptr;
```

Comment:
What's the motivation behind a static QgsMapSettings object? Why not just create one per map render?